### ISSUE TITLE
 Recieved network response for requestId 158.251 with no matching request.

### ISSUE THREAD

**Issue by kikyoar** [sitespeedio/sitespeed.io#1316]
hi,can U help me!

root@ubuntu:~# docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:4.0.4 --preScript login.js -b chrome http://www.ctnma.cn/ioop-bcs-web/main.do
Google Chrome 54.0.2840.71 
Mozilla Firefox 49.0.2
Starting Xvfb ...
[2016-11-07 09:28:15] Versions OS: linux 4.2.0-27-generic nodejs: v6.9.1 sitespeed.io: 4.0.4 browsertime: 1.0.0-beta.9 coach: 0.28.4
[2016-11-07 09:28:15] Starting chrome for analysing http://www.ctnma.cn/ioop-bcs-web/main.do 3 time(s)
[2016-11-07 09:28:15] Testing url http://www.ctnma.cn/ioop-bcs-web/main.do run 1
[2016-11-07 09:28:24] **Recieved network response for requestId 158.251 with no matching request.**
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Recieved network data for requestId 158.251 with no matching request.
[2016-11-07 09:28:24] Network loading finished for requestId 158.251 with no matching request.

Sometimes it is normal, sometimes reported this error！

**Comment by tobli** [comment]
Hi @kikyoar!
TLDR:
You can disregard this output, it's perfectly harmless and normal.

Background:
We create HAR files from Chrome by tapping in to the network event log from Chrome. Sometimes Chrome reports data for an earlier load (e.g. a slow image or an analytics request) when we start collecting data from the tested page. Since you're using a pre-script to do login, the stray request comes from that page load. We'll see if we can make the logging more clear/less verbose in the future.

I'm closing this, but please open an issue if the generated har files seems incorrect. Thanks!
/Tobias
